evalml/model_understanding/force_plots.py

```diff
@@ -0,0 +1,100 @@
+import numpy as np
+import shap
+
+from evalml.model_understanding.prediction_explanations import (
+    explain_predictions
+)
+
+
+def graph_force_plot(pipeline, rows_to_explain, training_data, y, matplotlib=False):
+    """ Function to generate force plots for the desired rows of the training data.
+
+    Args:
+        pipeline (PipelineBase): The pipeline to generate the force plot for.
+        rows_to_explain (list(int)): A list of the indices indicating which of the rows of
+            the training_data to explain.
+        training_data (pandas.DataFrame): The data used to train the pipeline.
+        y (pandas.Series): The target data for the pipeline.
+        matplotlib (bool): flag to display the force plot using matplotlib (outside of jupyter)
+            Defaults to False.
+
+    Returns:
+        list(dict(shap.AdditiveForceVisualizer)): The same as force_plot(), but with an additional
+            key in each dictionary for the plot.
+    """
+
+    def gen_force_plot(shap_values, training_data, expected_value, matplotlib):
+        """ Helper function to generate a single force plot. """
+        # Ensure there are as many features as shap values.
+        assert training_data.shape[1] == len(shap_values)
+
+        # TODO: Update this to make the force plot display multi-row array visualizer.
```

Comment:
Do we have an issue filed for this TODO?

Reply:
@dsherry yup, we traiged this : https://github.com/alteryx/evalml/issues/2287